oshc/main/regbackend.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth import get_user_model
+
+
+class EmailBackend(object):
+    def authenticate(self, username=None, password=None):
```

Comment:
The `authenticate` method should `return None` when the authentication credentials are invalid. Your `authenticate` function returns None implicitly. The intention of your code is much clearer with an explicit `return None`. :)